project.clj

```diff
@@ -39,6 +39,7 @@
                                org.apache.httpcomponents/httpclient
                                net.sourceforge.nekohtml/nekohtml
                                ring/ring-core]]
+                 [com.cemerick/url "0.1.1"]                           ; URL library
```

Comment:
I'm also using the `url` method, which resolves relative paths against a base url. e.x. `(str (url "http://localhost:3000/", "/api/public/oembed"))` returns `http://localhost:3000/api/public/oembed` whereas simple string concatenation would duplicate the `http://localhost:3000//api/public/oembed` (note the double `//`) which our router doesn't match.

If you'd rather not include the dependency we could just trim the trailing or leading "/", but I'm not sure if there are any other edge cases I'm missing.